docs/releasenotes.md

```diff
@@ -7,7 +7,30 @@ title: Haal Centraal BRP Personen Bevragen
 
 ## Versie 2.0.0
 
-De property reisdocumenten is verwijderd uit "Persoon".
+- het request is gewijzigd van GET naar POST, waarbij alle parameters verplaatst zijn van de querystring in de url naar de requestbody
+- requestbody is polymorf gedefinieerd, waarbij in parameter 'type' bepaalt welke parameters gebruikt mogen en moeten worden
+- parameter 'fields' is verplicht (required)
+- in het antwoord zitten geen json-HAL elementen \_links en \_embedded
+- in het antwoord is veld 'ingeschrevenPersonen' hernoemd naar 'personen'
+- het antwoord op elke zoekvraag (elk request anders dan raadplegen op burgerservicenummer) bevat maximaal een beperkte set gegevens van de persoon
+- veld 'reisdocumenten' is verwijderd
+- alle datums zijn polymorf gedefinieerd: nieuw veld type geeft aan welke velden voor de datum verwacht kunnen worden
+- alle datums hebben nieuw veld 'langFormaat', die de datum levert in voor mensen leesbare vorm
```

Comment:
"waarmee de datum in volledig uitgeschreven formaat wordt geleverd"  . een datum met alleen cijfers is ook voor mensen leesbaar.

Reply:
aangepast naar: waarmee de datum in volledig uitgeschreven formaat wordt geleverd, die direct aan eindgebruikers gepresenteerd kan worden